.travis.yml

```diff
@@ -203,7 +203,7 @@ before_install:
   fi
 - |-
   # Install autotools if asked for
-  if [[ "$TRAVIS_USE_AUTOTOOLS" == "1" ]]; then
+  if [[ "$TRAVIS_BUILD_SYSTEM" == "autotools" ]]; then
```

Comment:
Funny. Fixing this [has led](https://travis-ci.org/github/zenorogue/hyperrogue/jobs/691015281) MacOS-autotools build failing because these are apparently pre-installed.